config/prowgen/pkg/context.go

```diff
@@ -116,18 +113,75 @@ func (pc *ProwContext) Periodics(job *Job, periodicityHours int) {
 				BaseRef: pc.Branch,
 			},
 		},
-		Interval: strconv.Itoa(periodicityHours) + "h",
-		// TODO: use Cron instead of Interval
-		// Cron: pc.cronSchedule(periodicityHours),
+		PeriodicityHours: periodicityHours,
+		// Use a minute and startHour of 0 when adding the period here, but in JobFile
+		// when we actually generate the JobFile struct we'll recalculate values
+		// to spread the periodics evenly across every hour / day
+		Cron: cronSchedule(0, 0, periodicityHours),
```

Comment:
Can we leave the `Cron` value empty here? So it is clear that this value is not used.